internal/handler/command.go

```diff
@@ -19,12 +20,22 @@ import (
 	"github.com/edgexfoundry/device-sdk-go/internal/common"
 	"github.com/edgexfoundry/device-sdk-go/internal/transformer"
 	dsModels "github.com/edgexfoundry/device-sdk-go/pkg/models"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients"
 	contract "github.com/edgexfoundry/go-mod-core-contracts/models"
+
+	"github.com/ugorji/go/codec"
 )
 
 // Note, every HTTP request to ServeHTTP is made in a separate goroutine, which
 // means care needs to be taken with respect to shared data accessed through *Server.
-func CommandHandler(vars map[string]string, body string, method string, queryParams string) (*dsModels.Event, common.AppError) {
+func CommandHandler(
+	vars map[string]string,
+	body string,
+	method string,
+	queryParams string,
+	r *http.Request) (*dsModels.Event, common.AppError) {
```

Comment:
If we are passing the request( `r *http.Request) and into this function and down to other functions/methods, can we eliminate the other arguments since they can be obtained from the request?

Reply:
Excellent point, and one that I had initially (before submitting this `PR`) contemplated implementing, but chose not to so as to minimize the scope of the updates. But the point is valid, and confirms its value. I will make it so ✅